### PR TITLE
add netcdftime

### PR DESCRIPTION
**Pull request by jhamman**
cc @jswitt and @ocefpaf

### CODE REVIEW

**Comment by jhamman** [comment]
@ocefpaf - can I defer to you on the failed appveyor build?

File: recipes/netcdftime/COPYING [hunk 1]
```diff
@@ -0,0 +1,39 @@
+copyright: 2008 by Jeffrey Whitaker.
```
Comment (ocefpaf) [inline recipes/netcdftime/COPYING]: This is in the source tarball. I don't think we need to add it here.

**Comment by jhamman** [comment]
Thanks @ocefpaf - sorry for messing up your last commit. We're all green now!

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf - sorry for messing up your last commit. We're all green now!

No problem. I should not be editing like :-)

**Pull request merged**